### DISCUSSION TITLE
Feature Request: Curved Lines

### DISCUSSION THREAD

**Discussion by ice-cap0** [react-native-maps/react-native-maps#4596] [Feature requests]
### Summary

[Same feature request by different OP](https://github.com/react-native-maps/react-native-maps/issues/4068) was previously closed because it was created as a `bug` and the maintainers instructed OP to publish issue with Feature Request issue template. 
The Feature Request issue template doesn't seem to exist (?), so posting here.

Would you consider adding an option for curved polylines?

### Reproducible sample code

```TSX
n/a
```

### Steps to reproduce

n/a

### Expected result

n/a

### Actual result

n/a

### React Native Maps Version

any

### What platforms are you seeing the problem on?

iOS (Google Maps)

### React Native Version

any

### What version of Expo are you using?

Not using Expo

### Device(s)

any

**Comment by monholm** [comment]
The feature request template is placed in the discussion forum 👍
